.travis.yml

```diff
@@ -1,36 +0,0 @@
-os: linux
```

Comment:
If we are no longer using travis, please make sure to update README and a couple python files where we still reference travis, to make the entire CI clean.